ffwd-client/src/main/java/com/spotify/ffwd/FastForward.java

```diff
@@ -74,15 +96,26 @@ private FastForward(InetAddress addr, int port, DatagramSocket socket) {
     this.socket = socket;
   }
 
+  protected FastForward() throws UnknownHostException, SocketException {
+    this.addr = InetAddress.getByName(DEFAULT_HOST);
+    this.port = DEFAULT_PORT;
+    this.socket = new DatagramSocket();
+  }
+
+
   public void send(Metric metric) throws IOException {
-    sendFrame(metric.serialize());
```

Comment:
should both of these send functions just be combined into one that takes in the version?

Reply:
Are you thinking about something like this send(Metric metric , Version v1)  ?if not,  Give me the signature of the combine function you have in mind.

Reply:
I guess I'm a little confused by both send functions having the same name but maybe that's how it needs to be written and I'm missing something. was thinking something like `public void send(Metric metric, version)` but since it's two different types of metrics, maybe we can just be explicit with the names if possible? 
```
  public void sendV0(Metric metric) throws IOException {
    sendFrame(metric.serialize(), Version.V0);
  }

  public void sendV1(com.spotify.ffwd.v1.Metric metric) throws IOException {
    sendFrame(metric.serialize(), Version.V1);
  }
```

Reply:
Ok, I see your point now. We are not using a different name because we don't want to change  ForwardClient public interface.  With different names such as sendV1 or sendV0, users will have to make a code change whether they are using distribution or not. This is something we want to avoid. 
With the overload approach (using the same name),  you don't have to change anything if you are not using distribution.

Reply:
Okay cool, thank you for clarifying :)

Reply:
Great! can you merge it for me.

Reply:
I'll get you permissions 👍

Reply:
You should be able to merge now :)